README.md

```diff
@@ -1,18 +1,18 @@
-# Filecoin Station
+# Checker
 
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](https://protocol.ai/)
 [![](https://img.shields.io/badge/project-Filecoin-blue.svg?style=flat-square)](https://filecoin.io/)
-[![ci](https://github.com/filecoin-station/desktop/actions/workflows/ci.yml/badge.svg)](https://github.com/filecoin-station/desktop/actions/workflows/ci.yml)
+[![ci](https://github.com/CheckerNetwork/app/actions/workflows/ci.yml/badge.svg)](https://github.com/CheckerNetwork/app/actions/workflows/ci.yml)
 
-Filecoin Station is a desktop app that connects your computer’s idle resources
+Checker is a desktop app that connects your computer’s idle resources
 to the Filecoin network and rewards you with Filecoin. Taking part is simple,
 just launch the app and start earning. Learn more at
-[filstation.app](https://filstation.app).
+[app.checker.network](https://app.checker.network).
```

Comment:
TODO: https://app.checker.network/ redirects to filstation.app, which is confusing

Reply:
There is now https://docs.checker.network/running-a-checker

Reply:
app.checker.network now points to https://docs.checker.network/running-a-checker, waiting for DNS to propagate